R/general_lineplot.R

```diff
@@ -115,11 +110,7 @@ general_lineplot <- function(
       filter(AVAL > 0)
   }
 
-  time <- if (time_scale == "By Cycle") {
-    "ARRLT"
-  } else {
-    "AFRLT"
-  }
+  time <- switch(time_scale, `By Cycle` = "ARRLT", "AFRLT")
```

Comment:
suggestion: This is marginal, but simple `if else` statement fits better here.
```suggestion
  time <- if (time_scale == "By Cycle") "ARRLT" else "AFRLT"
```